src/project/internal/audacityproject.cpp

```diff
@@ -6,6 +6,8 @@
 #include "global/io/ioretcodes.h"
 
 #include "log.h"
+#include "global/io/fileinfo.h"
```

Comment:
Please follow the include order as described in https://github.com/musescore/muse_framework/wiki/CodeGuidelines